FlightReservation/src/screens/SignUp.js

```diff
@@ -82,33 +83,15 @@ const SignUp = function ({navigation}) {
                     paddingBottom: 40,
                   }}
                 />
-                <Pressable
-                  disabled={!formProps.isValid || loading}
-                  onPress={formProps.handleSubmit}>
-                  <View style={style.btn}>
-                    <Text style={{color: colors.white, fontSize: 18}}>
-                      Sign Up
-                    </Text>
-                  </View>
-                </Pressable>
-
+                <ButtonForm isValid = {formProps.isValid} handleSubmit = {formProps.handleSubmit} text = "Sign up" loading = {loading}/>
                 <Text
                   style={[
                     style.lower_content_text,
                     {paddingTop: 10, paddingBottom: 10},
                   ]}>
```

Comment:
avoid inline styles

Reply:
We've fixed all the inline styles in the Sign up form and ButtonForm component.